bobtemplates/plone/buildout/buildout.cfg.bob

```diff
@@ -42,4 +42,15 @@ recipe = zc.recipe.testrunner
 defaults = ['--auto-color', '--auto-progress']
 eggs =
     Plone
-    ${buildout:eggs}
\ No newline at end of file
+    ${buildout:eggs}
+
+[zopepy]
```

Comment:
In order to reduce confusion, I would name either name the section ``plone-helper-scripts`` or similar.

Reply:
regarding the zopepy it is useful for using autocompletion in editors like sublime text or vs code.
If you set the Python interpreter for the editor, your editor will have all buildout packages and do autocompletion. @jensens if you know a better generic way to do archive this, i'm happy to here it.

Reply:
naming the part: plone-helper-scripts is a good idea, let's do it.